wezterm-gui/src/termwindow/mod.rs

```diff
@@ -363,13 +363,15 @@ pub struct TermWindow {
     /// Window dimensions and dpi
     pub dimensions: Dimensions,
     pub window_state: WindowState,
+    pub resizes_pending: usize,
     /// Terminal dimensions
     terminal_size: TerminalSize,
     pub mux_window_id: MuxWindowId,
     pub mux_window_id_for_subscriptions: Arc<Mutex<MuxWindowId>>,
     pub render_metrics: RenderMetrics,
     render_state: Option<RenderState>,
     input_map: InputMap,
+    pending_scale_changes: LinkedList<resize::ScaleChange>,
```

Comment:
let's put this alongside `resizes_pending` so that it is easier to see the related fields at a glance